### PR TITLE
Added documentation about Securty => userNameIsEmail

### PR DESCRIPTION
**Pull request by JeffreyPerplex**
Added documentation about Securty => userNameIsEmail

### CODE REVIEW

File: Reference/Config/umbracoSettings/index.md [hunk 1]
```diff
@@ -287,6 +290,9 @@ option to "false" and thereby enabling it.
 As stated in the comment above, this setting was introduced in v4.8. If it's set to true it's not possible to see disabled users, which means it's
 not possible to re-enable their access to the back office again. It also means you can't create an identical username if the user was disabled by a mistake.
 
+**`<usernameIsEmail>`**
+This settings specifies whether the username and email are seperate fields in the backoffice editor. When set to false, you can specify an emailaddress and loginname, and only the loginname can be used to log on. When set the true (the default value) the loginname is always the same as the emailaddress.
```
Comment (Jeavon) [inline Reference/Config/umbracoSettings/index.md]: I think loginname should always be username?

Comment (Jeavon) [inline Reference/Config/umbracoSettings/index.md]: "settings" should be singular "setting"

Comment (Jeavon) [inline Reference/Config/umbracoSettings/index.md]: "emailaddress" should be two words "email address"

**Comment by JeffreyPerplex** [comment]
Hi @Jeavon, thanks for the quick feedback and checking my stuff. Do I need to merge these in and commit them again, or how does that work?

Thanks for helping with the documentation #h5yr!

**Comment by Jeavon** [comment]
@JeffreyPerplex if you make an additional commit to your fork/branch "JeffreyPerplex:patch-1" then this PR will update

**Comment by JeffreyPerplex** [comment]
Check, updated!

**Comment by Jeavon** [comment]
Perfect, many thanks for contributing to the docs!

**Pull request merged**